tutorials/29_Serializing_Pipelines.ipynb

```diff
@@ -0,0 +1,413 @@
+{
```

Comment:
<ul><li>We can mention more use cases, like "you can save on disk, database, or send over the network".</li><li>We can make it clear that editing the YAML format is expected, like "Haystack uses YAML to serialize pipelines so that it's easy for humans to make changes without the need to go back and forth with Python code. Haystack explicitly supports making changes to a serialized pipeline before going back to Python to run it. In this tutorial, we'll..."</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-tutorials/pull/272/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>